### PR TITLE
Normalize objective functions

### PR DESCRIPTION
**Pull request by vondele**
go from sum of squared error to root mean squared error in the score, divide log prob by the number of positions (i.e. use geometric mean)

### CODE REVIEW

**Comment by vondele** [comment]
should be non-functional apart from numerics.

**Comment by robertnurnberg** [comment]
Instead of manually added to counter, could just read of the lengths of the data containers?

Btw: I am working on a major rewrite of the internal data structures, going full numpy. Hope is for cleaner and clearer code that will also be significantly faster. Stay tuned...

File: scoreWDL.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import argparse, json, matplotlib.pyplot as plt, numpy as np, time
+import argparse, json, matplotlib.pyplot as plt, numpy as np, time, math
```
Comment (robertnurnberg) [inline scoreWDL.py:1]: Using `np.sqrt` is also a possibility. ;)

**Comment by robertnurnberg** [comment]
Of course, I agree with the idea behind the pr.

**Pull request merged**